test/metabase/query_processor/dashboard_test.clj

```diff
@@ -403,3 +405,17 @@
                                                         :type    "category"
                                                         :value   nil
                                                         :default ["Gizmo"]}])))))))))
+
+(deftest running-a-query-for-dashcard-sets-last-viewed-at
+  (binding [tu.thread-local/*thread-local* false]
```

Comment:
```suggestion
  (mt/test-helpers-set-global-values!
```

I don't think that dynamic var is meant to be bound manually, we should use the macro instead

Reply:
Oops, you're right - thanks!